Test/Framework/Graphics/GraphicsDeviceTestFixtureBase.cs

```diff
@@ -51,6 +51,11 @@ internal class GraphicsDeviceTestFixtureBase
         [SetUp]
         public virtual void SetUp()
         {
+#if !XNA
+            // We enable the half-pixel offset for XNA compatibility
+            GraphicsDeviceManager.UseStandardPixelAddressing = false;
```

Comment:
if SpriteBatch rendering is not affected by the flag and produces the same results, then we don't need that. Shouldn't we instead test MG with the default setting?

Reply:
The tests that failed were not SpriteBatch tests. I don't really know why they did fail though.

 But we do want to set this flag in the tests for when we add tests for the built in effects (if we ever get around to that). Of course DX should support manipulating half pixel offset at runtime for it to work.